SERVER.md

```diff
@@ -44,15 +44,15 @@ server:
           session_duration: "8h"
           access_key_id: "123ABC..."
           secret_access_key: "ZXY098..."
-      sean:
```

Comment:
This was here to imply that you might have separate environments for each developer and that each of these would be a target.  Fine to remove it, but just wanted to clarify the original intention.

Reply:
I think we have enough there to make it clear that you can create as many of these as you need with whatever names you want. If we think that is an interesting usecase to draw attention to, lets' actually describe it in the README.

Reply:
agreed - fine to strike it